content/en/docs/ambient/usage/extensibility/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Extensibility
```

Comment:
For now, I would just make this page be "Extend waypoints with WebAssembly plugins" and have it be at `/docs/ambient/usage/extend-waypoint-wasm/`'.

(Make sure the weight is such that the doc ends up after the waypoint docs but before the debugging docs)